GitHub-Test-4-PyNLPDiscuss/JordanJ-GitHubTest4.md

```diff
@@ -0,0 +1,14 @@
+# GitHub Test 4 
+
+
+This is my response from the reading 
+
+1. Based off of the readings I found it interesting that sorting and creating patterns can help people remember. I'm curious about what the limits of matching and patterns are in NLP. One area that I am interested in Sentiment Analysis in the context of determining whether data is positive, negative or neutral, this keeps tracks of textural data  and allow businesses to track and analyze customers responses.
```

Comment:
You did a great job on making sure you gave a full review about the reading we had for class! Great job.